<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab_Edit</class>
 <widget class="QWidget" name="Tab_Edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="4">
    <widget class="QListWidget" name="list_opt"/>
   </item>
   <item row="0" column="4" rowspan="9">
    <widget class="QTextEdit" name="edit_edit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>&amp;Template</string>
     </property>
     <property name="buddy">
      <cstring>combo_template</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="push_add">
     <property name="text">
      <string>Add..</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="push_remove">
     <property name="text">
      <string>Remove...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="push_help">
     <property name="text">
      <string>&amp;Help</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QComboBox" name="combo_template">
     <item>
      <property name="text">
       <string>Launch Script</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>INCAR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>POTCAR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>KPOINTS</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>user1:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>user2:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>user3:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>user4:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLineEdit" name="edit_user1"/>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QLineEdit" name="edit_user2"/>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QLineEdit" name="edit_user3"/>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QLineEdit" name="edit_user4"/>
   </item>
   <item row="0" column="6" rowspan="9">
    <widget class="QListWidget" name="list_POTCARs"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&amp;Optimization Type</string>
     </property>
     <property name="buddy">
      <cstring>combo_optType</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QComboBox" name="combo_optType">
     <item>
      <property name="text">
       <string>VASP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GULP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PWscf</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>combo_optType</tabstop>
  <tabstop>combo_template</tabstop>
  <tabstop>list_opt</tabstop>
  <tabstop>push_add</tabstop>
  <tabstop>push_remove</tabstop>
  <tabstop>push_help</tabstop>
  <tabstop>edit_user1</tabstop>
  <tabstop>edit_user2</tabstop>
  <tabstop>edit_user3</tabstop>
  <tabstop>edit_user4</tabstop>
  <tabstop>edit_edit</tabstop>
  <tabstop>list_POTCARs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
